app/javascript/modules/listings/ListingDirectory.scss

```diff
@@ -22,6 +22,10 @@
   margin-bottom: 0.4rem;
 }
 
+.is-card-link::after {
+  content: none !important;
+}
+
```

Comment:
question: is there any plan to update the [bloom component](https://github.com/bloom-housing/ui-components/blob/main/src/page_components/listing/ListingCard.scss#L74), or is this behavior override considered a permanent solution?

Reply:
Making the entire Card interactive appears to have been a deliberate decision, so I wasn't sure. I have messaged the Bloom Engineers seeing if this is something that we can change on the Bloom side, or at least make optional.  In the meantime I created this override so this ticket could be completed